test/box-luatest/gh_9923_invalid_iproto_index_name_mapping_on_the_server_side_test.lua

```diff
@@ -0,0 +1,75 @@
+local net = require('net.box')
+local server = require('luatest.server')
+local t = require('luatest')
+
+local g = t.group()
+
+g.before_all = function(cg)
+    cg.server = server:new({alias = 'master'})
+    cg.server:start()
+    cg.server:exec(function()
+        box.schema.create_space('test', {
+            format = {
+                {name = "id", type = "unsigned"},
+                {name = "name", type = "string"},
+            },
+        })
+
+        box.space.test:create_index('primary', {
+            type = 'tree',
+            unique = true,
+            parts = {1, 'unsigned'},
+        })
+
+        box.space.test:create_index('secondary', {
+            id = 5,
+            type = 'tree',
+            unique = true,
+            parts = {2, 'string'},
+        })
+    end)
+
+    -- fetch schema = false to resolve index on server instead of client
+    cg.conn = net.connect(cg.server.net_box_uri, {fetch_schema = false})
+    t.assert_is_not(cg.conn, nil)
+end
+
+g.after_all = function(cg)
+    cg.server:drop()
+end
+
+g.before_each = function(cg)
+    cg.server:exec(function()
+        box.space.test:truncate()
+    end)
+end
+
+g.after_each = function(cg)
```

Comment:
These hooks are not actually executed because they are defined in this way:
```lua
g.after_each = function() ... end
```

Please, define them in this way:
```lua
g.after_each(function() ... end)
```

Applicable to all hooks: `before_all`, `after_all`, `before_each`, `after_each`.